scalding-core/src/main/scala/com/twitter/scalding/TypedPipeChecker.scala

```diff
@@ -0,0 +1,32 @@
+package com.twitter.scalding
+
+/**
+ * This class is used to assist with testing a TypedPipe
+ */
+object TypedPipeChecker {
+
+  /*
+   * Takes a List and a transform function.
+   * The resulting TypedPipe form the transform will be run through asserts
+   */
+  def checkOutputTransform[T, U](input: List[T])(transform: TypedPipe[T] => TypedPipe[U])(assertions: List[U] => Unit) =
```

Comment:
can we put a return type on all methods?

also, what about mre general:
```scala
def checkOutputTransform[T, U, R](input: List[T])(transform: TypedPipe[T] => TypedPipe[U])(assertions: List[U] => R): R =
```
then you could use methods that return other than unit.